nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -10,14 +10,13 @@ in
   # `pandoc lemmy.md -t docbook --top-level-division=chapter --extract-media=media -f markdown+smart > lemmy.xml`
   meta.doc = ./lemmy.xml;
 
-  options.services.lemmy = {
+  imports = [
+    (mkRemovedOptionModule [ "services" "lemmy" "jwtSecretPath" ] "This option is no longer needed, the secret gets generated automatically now.")
```

Comment:
```suggestion
    (mkRemovedOptionModule [ "services" "lemmy" "jwtSecretPath" ] "This option is no longer required because the secret gets generated automatically.")
```